doc/deprecation-process.md

```diff
@@ -118,25 +104,6 @@ For additional support, open a new issue in the [Issues](https://github.com/micr
 
 ### Source Code Deprecation Attributes
 
-Add `#[deprecated]` attributes to all public APIs in the crate to provide compile-time warnings to users:
```

Comment:
I wouldn't delete these entirely. I agree that putting the module-level first is best because this doc appears to be more about deprecating a crate, but maybe should at least document how to deprecate types/methods. I'd move them to after and maybe just have one or two examples and link to <https://doc.rust-lang.org/reference/attributes/diagnostics.html#r-attributes.diagnostics.deprecated> for more information.